diff --git a/include/configs/versatile.h b/include/configs/versatile.h
index de23375..b3b7ac9 100644
--- a/include/configs/versatile.h
+++ b/include/configs/versatile.h
@@ -22,6 +22,16 @@
 #define CONFIG_VERSATILE	1	/* This is Versatile Platform Board	*/
 #define CONFIG_ARCH_VERSATILE	1	/* Specifically, a Versatile	*/
 
+#define CONFIG_ARCH_VERSATILE_QEMU
+#define CONFIG_INITRD_TAG
+#define CONFIG_BOOTCOMMAND \
+  "sete ipaddr IP_ADDR;"\
+  "sete serverip ROUTE_ADDR;"\
+  "set bootargs 'route=ROUTE_ADDR console=tty0 console=ttyAMA0 root=/dev/ram';"\
+  "tftpboot 0x00007fc0 uImage;"\
+  "tftpboot 0x00807fc0 rootfs.cpio.uboot;"\
+  "bootm 0x7fc0 0x807fc0"
+
 #define CONFIG_SYS_MEMTEST_START	0x100000
 #define CONFIG_SYS_MEMTEST_END		0x10000000
 
@@ -106,7 +116,7 @@
  * Miscellaneous configurable options
  */
 #define CONFIG_SYS_LONGHELP	/* undef to save memory */
-#define CONFIG_SYS_CBSIZE	256		/* Console I/O Buffer Size */
+#define CONFIG_SYS_CBSIZE	1024		/* Console I/O Buffer Size */
 /* Monitor Command Prompt	 */
 #ifdef CONFIG_ARCH_VERSATILE_AB
 # define CONFIG_SYS_PROMPT	"VersatileAB # "
